libs/core/src/scss/interaction-state/_hover.scss

```diff
@@ -0,0 +1,54 @@
+@use 'sass:color';
+@use 'sass:meta';
+@use '../utils';
+
+// TODO: Accessibility - Loudness must respect user preferences
+// See https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast
+// TODO: Await UX decision on default loudness and update
+$default-loudness: utils.get-loudness('loud');
+
+@mixin apply-hover($background-color, $color, $property, $loudness: null) {
+  @if meta.type-of($background-color) != color {
+    @error "$background-color must be a valid CSS color value. See https://developer.mozilla.org/en-US/docs/Web/CSS/color_value";
+  }
+
+  @if meta.type-of($color) != color {
+    @error "$color must be a valid CSS color value. See https://developer.mozilla.org/en-US/docs/Web/CSS/color_value";
+  }
+
+  // ---------------------------------------------------------------------------
+  // Determine if lightness should increase or decrease.
+  // The default is increase (a positive percentage).
+  //
+  // When background color is transparent assume that the canvas underneath is
+  // a light color. We cannot really determine if lightness should increase or
+  // decrease, but this is a best guess. It's possible to work around by passing
+  // a $background-color (or $color) parameter with a value other than
+  // transparent.
+  //
+  // Sidenote: The color transparent is defined as black with 0% opacity, i.e.,
+  // 100% transparent. Often declared as rgba(0, 0, 0, 0%) in browsers.
+  // ---------------------------------------------------------------------------
+  $relative-lightness: $default-loudness !default;
+
+  @if $loudness {
+    $relative-lightness: $loudness;
+  }
+
+  @if color.lightness($background-color) > color.lightness($color) {
+    // Decrease relative lightness
```

Comment:
```suggestion
    // Reverse relative lightness
```

Reply:
If i understand it correctly 🤷

Reply:
Hmm. Technically it is "reversed" as in "negate" or "flip the positive/negative sign". But it is supposed to tell the reader that "if background color is lighter than foreground color then REDUCE lightness". Reduce lightness = decrease relative lightness = make it darker.

I don't think "Reverse" is a correct term for this. Do you have other suggestions?

Reply:
Perhaps the comment should just be: 

`// Make it darker`

😁